scarlet/wavelet.py

```diff
@@ -0,0 +1,300 @@
+import autograd.numpy as np
+from . import fft
+from . import Cache
+from scipy.stats import median_absolute_deviation as mad
+
+# Filter for the scarlet transform. Here bspline
+h = np.array([1. / 16, 1. / 4, 3. / 8, 1. / 4, 1. / 16])
+
+class Starlet(object):
+    """ A class used to create the Wavelet transform of a cube of images from the 'a trou' algorithm.
+
+        The transform is performed by convolving the image by a seed starlet: the transform of an all-zero
+        image with its central pixel set to one. This requires 2-fold padding of the image and an odd pad
+        shape. The fft of the seed starlet is cached so that it can be reused in the transform of other
+        images that have the same shape.
+    """
+    def __init__(self, image, lvl = None, starlet = None, direct = False):
+        """ Initialise the Starlet object
+
+        Paramters
+        image: Fourier object
```

Comment:
This type seems to be an ordinary array, not an instance of `Fourier`

Reply:
Yup, old implementation, sorry